aws/resource_aws_ram_resource_share_accepter.go

```diff
@@ -128,33 +130,51 @@ func resourceAwsRamResourceShareAccepterCreate(d *schema.ResourceData, meta inte
 }
 
 func resourceAwsRamResourceShareAccepterRead(d *schema.ResourceData, meta interface{}) error {
-	conn := meta.(*AWSClient).ramconn
 
-	invitation, err := resourceAwsRamResourceShareGetInvitation(conn, d.Id(), ram.ResourceShareInvitationStatusAccepted)
+	client := meta.(*AWSClient)
+	conn := client.ramconn
 
-	if err == nil && invitation == nil {
-		log.Printf("[WARN] No RAM resource share invitation by ARN (%s) found, removing from state", d.Id())
-		d.SetId("")
-		return nil
+	invitation, err := resourceAwsRamResourceShareGetInvitation(client, d.Id(), ram.ResourceShareInvitationStatusAccepted)
+
+	if err != nil {
+		return fmt.Errorf("Error retrieving invitation for resource share %s: %s", d.Id(), err)
+	}
+
+	if invitation != nil {
+		d.Set("invitation_arn", invitation.ResourceShareInvitationArn)
+		d.Set("receiver_account_id", invitation.ReceiverAccountId)
+	} else {
+		d.Set("receiver_account_id", meta.(*AWSClient).accountid)
```

Comment:
If invitation is still present, we source both attributes from the invitation.
If the invitation is not present anymore, we source only the receiver_account_id from the current client accountid.